tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "declaration": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "outDir": "dist"
```

Comment:
Do you commit `dist` on publish?

Reply:
Yes, see my comment below. Note that I plan to make the changes for releasing in a separate PR, but we'd need to commit the `dist` folder in order to be able to run this action.